ext/net/lib.deno_net.d.ts

```diff
@@ -450,5 +450,241 @@ declare namespace Deno {
     options?: StartTlsOptions,
   ): Promise<TlsConn>;
 
+  /** @category Network */
+  export interface QuicTransportOptions {
```

Comment:
All these typings are missing the `**UNSTABLE**: New API, yet to be vetted.` banners and `@experimental` markers. @crowlKats can double check if they are marked properly